app/src/main/java/com/nervousfish/nervousfish/modules/util/AUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.nervousfish.nervousfish.modules.util;
+
+/**
+ * This class should be extended by all platform-specific {@link IUtils} implementations and contains
+ * all cross-platform functionality
+ */
+@SuppressWarnings("PMD.AbstractClassWithoutAnyMethod")
+abstract class AUtils implements IUtils {
```

Comment:
Yes, but in the future we may add more utility functions to it. The alternative was to add a QRGenerator to the service locator, but when we get more utility classes the service locator will have a lot of methods. However, I just thought that some classes like the key generator are also what you could call "utility classes". Do you think that we should add a utility class as is done in this PR to the service locator and also refractor classes like th key generator to that class, or is it better to make every utility a separate entry in the service locator?